relay-server/src/services/buffer/envelope_store/sqlite.rs

```diff
@@ -426,9 +427,9 @@ fn extract_envelope(row: SqliteRow) -> Result<Box<Envelope>, SqliteEnvelopeStore
     let received_at: i64 = row
         .try_get("received_at")
         .map_err(SqliteEnvelopeStoreError::FetchError)?;
-    let start_time = StartTime::from_timestamp_millis(received_at as u64);
 
-    envelope.set_start_time(start_time.into_inner());
+    let received_at = ReceivedAt::from_timestamp_millis(received_at);
```

Comment:
We should be able to call `DateTime::from_timestamp_millis` directly here, and remove that function from the extractor interface.